### ISSUE TITLE
Warning output on import is being written out twice

### ISSUE THREAD

**Issue by wsmelton** [PowerShell/PowerShellForGitHub#34]
Anytime you import this module it is outputting the text multiple times:
```powershell
> ipmo PowerShellForGitHub
C:\Program Files\WindowsPowerShell\Modules\PowerShellForGitHub\0.1.0\ApiTokens.psm1 does not exist, skipping import
This module should define $global:gitHubApiToken with your GitHub API access token in ApiTokens.psm1. Create this file if it doesn't exist.
You can simply rename ApiTokensTemplate.psm1 to ApiTokens.psm1 and update value of $global:gitHubApiToken.
You can get GitHub token from https://github.com/settings/tokens
If you don't provide it, you can still use this module, but you will be limited to 60 queries per hour.
C:\Program Files\WindowsPowerShell\Modules\PowerShellForGitHub\0.1.0\ApiTokens.psm1 does not exist, skipping import
This module should define $global:gitHubApiToken with your GitHub API access token in ApiTokens.psm1. Create this file if it doesn't exist.
You can simply rename ApiTokensTemplate.psm1 to ApiTokens.psm1 and update value of $global:gitHubApiToken.
You can get GitHub token from https://github.com/settings/tokens
If you don't provide it, you can still use this module, but you will be limited to 60 queries per hour.
```

As well can you please remove this as information output and change to warning so we have the ability to ignore the output. Right now there is now way to hide this output from being sent to the console or command line.

**Comment by HowardWolosky** [comment]
This is being fixed as part of my re-work of authentication configuration.

**Comment by HowardWolosky** [comment]
Fixed in 0.2.0
